norse/torch/module/lift.py

```diff
@@ -0,0 +1,49 @@
+from typing import Tuple, Union
+
+
+import torch
+
+
+class Lift(torch.nn.Module):
+    """Lift applies a given torch.nn.Module over
+       a temporal sequence.
```

Comment:
I would maybe add something like `... In other words this module applies the given torch.nn.Module N times, where N is the outer dimension in the provided tensor`. Just to explain exactly what "applies" mean

Reply:
Done